packages/core/src/api/app/AppBuilder.tsx

```diff
@@ -108,6 +117,13 @@ export default class AppBuilder {
       rendered = <ApiProvider apis={this.apis} children={rendered} />;
     }
 
+    rendered = (
+      <FeatureFlagsContextProvider
```

Comment:
I think we should stick to just the APIs for for communicating with the app. If this was a commonly used pattern performance would be horrible. If there's a need to observe feature flags then build that into the API using Observables, but tbh I think our current reload strategy if fine for them.